frontend/src/components/task/task.js

```diff
@@ -669,13 +670,37 @@ class Task extends Component {
       </span>)
     }
 
+    const resendNewOrder = (e, itemPrice, userId, taskId) => {
```

Comment:
So here will be something like `getOrder` with the order id and it will return the `payment_url` from order model
https://github.com/worknenjoy/gitpay/blob/master/models/order.js

If this attribute is present, show the button that direct the user to that url again

Reply:
Understood.

Now, will look like this:

Two failed payments, but only shows the button for one, because the other doesn't have the payment_url in order.

![Screenshot from 2019-07-08 22-43-11](https://user-images.githubusercontent.com/21283066/60853303-d973ea00-a1d1-11e9-8079-1af810de30da.png)